README.md

```diff
@@ -1,2 +1,23 @@
 # lobster-ice
```

Comment:
Would you mind adding a section explaining database connection? I believe a new developer may choose from connecting to your MongoDB Atlas instances (which you should whitelist for them), or choose to switch to a local database (This is what I tried, feel free to leave as TODO item).

Reply:
Just added instructions for this